hw/ip/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -344,7 +344,11 @@ chars = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
           ''',
```

Comment:
Do you want to update the generic `alert_handler.hjson` as well?

Reply:
Thanks Sri for the reminder. I think generic `alert_handler.hjson` should be okay because the alert_i is not connected to any driver. So I do not think these exclusions are needed.

Reply:
Hey @cindychip, I would nevertheless update the generic version as well, just to keep it in sync. It can be confusing to if these generated files are out of sync...

Reply:
okay sounds good then, I will update the generic version then :)